server-common/src/main/java/org/apache/gravitino/server/authorization/jcasbin/JcasbinAuthorizer.java

```diff
@@ -529,14 +530,14 @@ private void loadPolicyByRoleEntity(RoleEntity roleEntity) {
               String.valueOf(roleEntity.id()),
               securableObject.type().name(),
               String.valueOf(MetadataIdConverter.getID(securableObject, metalake)),
-              privilege.name().name().toUpperCase(),
+              AuthorizationUtils.replaceLegacyPrivilegeName(privilege.name()).name().toUpperCase(),
```

Comment:
`toUpperCase` should specify the `Locale`.

Reply:
Fixed.